web/src/buddy_list.js

```diff
@@ -1,16 +1,23 @@
 import $ from "jquery";
 
+import render_section_header from "../templates/buddy_list/section_header.hbs";
```

Comment:
This commit ("buddy list: Add user count and link to see full user list.") would ideally be separated into three commits:

* The refactor to get the data object
* Making the template for the headers and putting the user count in it
* Adding the links

I can do this in a bit.

Reply:
Done!